includes/Core/Email_Reporting/templates/email-report/parts/conversions-timeline.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Conversions timeline image part.
+ *
+ * Renders a timeline image based on whether the change is positive or negative.
+ *
+ * @package   Google\Site_Kit\Core\Email_Reporting
+ * @copyright 2025 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ *
+ * @var float    $change        The percentage change value.
+ * @var callable $get_asset_url Function to get asset URLs.
+ */
+
+$is_positive = (float) $change >= 0;
+$image_name  = $is_positive ? 'conversions-timeline-green.png' : 'conversions-timeline-red.png';
+$image_url   = $get_asset_url( $image_name );
```

Comment:
Lets use ternary check directly within get assets function, to signal that value is hardcoded string and does not need escaping/validation
```suggestion
$image_url   = $get_asset_url( $is_positive ? 'conversions-timeline-green.png' : 'conversions-timeline-red.png' );
```